runtime/src/main/java/org/corfudb/runtime/collections/RocksDbEntryIterator.java

```diff
@@ -0,0 +1,84 @@
+package org.corfudb.runtime.collections;
+
+import io.netty.buffer.Unpooled;
+import org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuError;
+import org.corfudb.util.serializer.ISerializer;
+import org.rocksdb.RocksDBException;
+import org.rocksdb.RocksIterator;
+
+import java.util.AbstractMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.NoSuchElementException;
+
+/**
+ *
+ * Created by Maithem on 1/21/20.
+ */
+
```

Comment:
Please remove this and add a proper JavaDoc comment.

Reply:
Done.